plenary/plenary-35.yaml

```diff
@@ -0,0 +1,325 @@
+        message: considering no progress for the development of ISO 9735-11 since it was registered as PWI in 2013
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2016-10-21
+        message: resolves to withdraw ISO/PWI 9735-11
+
+
+  - category:
+    dates:
+      - 2016-10-21
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Establishment of ISO/TC 154/WG7"
+    identifier: 2016-10
+    considerations:
+      - type: considering
+        date_effective: 2016-10-21
+        message: considering that ISO/NP 21981 ballot closed with approval of a new WG to be established by the TC
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2016-10-21
+        message: resolves to create ISO/TC 154/WG 7 with the title "Business"
+
+
+  - category:
+    dates:
+      - 2016-10-21
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Appointment of Mr. Klaus-Dieter Naujok as convener of ISO/TC 154/WG7"
+    identifier: 2016-11
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2016-10-21
+        message: resolves to appoint Mr. Klaus-Dieter Naujok as the convener of Working Group 7, effective immediately
+
+
+  - category:
+    dates:
+      - 2016-10-21
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Allocation of the secretariat of ISO/TC 154/WG7 to SAC"
+    identifier: 2016-12
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2016-10-21
+        message: resolves that the secretariat of ISO/TC 154/WG 7 is allocated to SAC. In addition the ISO/TC154 secretariat is requested to initiate an appropriate call for experts for the new WG 7
+
+
+  - category:
+    dates:
+      - 2016-10-21
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Distribution of approved NWIP"
+    identifier: 2016-13
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: recommends
+        date_effective: 2016-10-21
+        message: recommends that all approved New Work Items Proposals from the MoU on eBusiness Management Group (MoU/MG) members shall be disseminated to the MoU/MG membership for review and possible action at the earliest stage
```

Comment:
In this case, I put `recommends` string value as `type`. Is it ok?

Reply:
Agree.